components/crud-web-apps/jupyter/backend/apps/common/yaml/spawner_ui_config.yaml

```diff
@@ -25,6 +25,8 @@ spawnerFormDefaults:
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-pytorch-cuda-full:master-1831e436
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-tensorflow-full:master-1831e436
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-tensorflow-cuda-full:master-1831e436
+    hideRegistry: false
```

Comment:
We will also need to apply these changes to the manifests files, but lets do this after the refactored config file